bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/db/JdbcPostgresqlDAO.java

```diff
@@ -200,8 +228,23 @@ public List<Column> doGetTableColumns(ItemsVO vo) throws JdbcSQLException {
      *************/
 
     /*
-     * Override since PostgreSQL does not support setting NOT NULL in the same clause as ALTER COLUMN .. TYPE
+     * Override since PostgreSQL does not support setting NOT NULL in the same
+     * clause as ALTER COLUMN .. TYPE
      */
+
+    @Override
+    public void doCreateItemTable(ItemVO vo) throws JdbcSQLException {
+        String sql = StringUtilsExt.replaceArrayMerge(this.sqlCreateItemTable,
+                new String[] { "#tableName#", "#dbType#", "#tablePrimaryKey#" },
+                new String[] { vo.getTableName(), vo.getDbType(), sqlTypes.get("tablePrimaryKey") });
+        logger.debug("JDBC::doCreateItemTable sql={}", sql);
+        try {
+            Yank.execute(sql, null);
+        } catch (YankSQLException e) {
+            throw new JdbcSQLException(e);
+        }
+    }
+
```

Comment:
It looks like this method doesn't differ from the base implementation, so there should be no reason to override it?

The comment above also refers to `doAlterTableColumn`, which is now below this method.

Reply:
It's the same case here, I added the override because the query it uses needed to be changed.